src/flash/nor/stm32h7x.c

```diff
@@ -804,6 +823,16 @@ static int stm32x_probe(struct flash_bank *bank)
 			/* flash size is 2M or 1M */
 			flash_size_in_kb /= 2;
 		break;
+	case 0x483:
+		/* For STM32H72x/73x
+		 *  - STM32H7xxxG devices contains single bank, 1024 Kbyte
+		 */
+		if (flash_size_in_kb == 128)
+			has_dual_bank = false;
+		else
+			/* flash size is 2M or 1M */
+			flash_size_in_kb /= 1;
+		break;
 	default:
```

Comment:
@galak This were my changes only! All the other changes are strange for me currently and comes from current master!

Reply:
Is there an upstream gerrit reference?